### PR TITLE
Add codepipline and simple examples

### PR DESCRIPTION
**Pull request by jmonte-sph**
- [x] codebuild from s3
- [x] codebuild for codepipline

### CODE REVIEW

**Review by niroz89 — COMMENTED** [review]
* Let's use the `terraform-aws-cicd` module itself in the complete example, once that's ready to be public though, meanwhile we can remove it.

* it's better not to include any real values.

File: examples/codepipeline/main.tf [hunk 1]
```diff
@@ -0,0 +1,99 @@
+locals {
+  repository_branch = "main"
+  repository_owner  = "SPHTech-Platform"
+  repository_name   = "terraform-aws-codebuild"
+  codestar_arn      = "arn:aws:codestar-connections:us-east-1:123456789012:connection/aEXAMPLE-8aad-4d5d-8878-dfcab0bc441f"
```
Comment (niroz89) [inline examples/codepipeline/main.tf:5]: provide dummy.

File: examples/codepipeline/main.tf [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+
+  codebuild_name                 = "test-build-project"
+  codebuild_output_artifact_name = "build_artifact"
+  artifact_bucket_name           = "test-project-artifact"
+
+
+  codepipeline_name = "test-build-pipeline"
+  kms_key_arn       = "arn:aws:kms:us-east-1:123456789012:key/1234aaaa-bbbb-cccc-dddd-abcdabcdab"
```
Comment (niroz89) [inline examples/codepipeline/main.tf:13]: provide dummy

File: examples/codepipeline/main.tf [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+  }
+
+}
+
+module "codebuild" {
+  source = "../../"
+
+  name                  = "build-project"
+  description           = "codebuild for test project"
+  build_image           = "aws/codebuild/standard:5.0"
```
Comment (niroz89) [inline examples/codepipeline/main.tf:85]:
```suggestion
  build_image           = "aws/codebuild/standard:6.0"
```

File: examples/codepipeline/main.tf (hunk 2 above)
Comment (jmonte-sph) [reply examples/codepipeline/main.tf:13]: this is a dummy value

File: examples/codepipeline/main.tf (hunk 1 above)
Comment (jmonte-sph) [reply examples/codepipeline/main.tf:5]: this is a dummy value

File: examples/codepipeline/main.tf (hunk 3 above)
Comment (jmonte-sph) [reply examples/codepipeline/main.tf:85]: This is is to show the usage of 5.0. Since 6.0 is recently released

**Review by niroz89 — APPROVED** [review]
Let's merge this first, later help to change the example to use the `terraform-aws-cicd` instead of resource pipeline.

**Pull request merged**